pkgs/misc/brightnessctl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
+  name = "brightnessctl-${version}";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/Hummer12007/brightnessctl/archive/0.3.2.tar.gz";
+    sha256 = "92687d8739b8395cf942287bdb8905a78b428102ab6029680829ab917363e0ab";
+  };
+
+  patchPhase = ''
+    substituteInPlace 90-brightnessctl.rules --replace /bin/ ${coreutils}/bin/
+    substituteInPlace 90-brightnessctl.rules --replace %k '*'
+  '';
+
+  installPhase = ''
```

Comment:
We should be able to reuse the install from the Makefile by setting "PREFIX=$(out)" in makeFlags. This is a little bit better practice especially for when renames, new files, etc are introduced.